resources/terraform/mainnet/main.tf

```diff
@@ -60,7 +60,7 @@ module "mainnet" {
     regions            = var.aws_region
     instance-count     = var.instance_count["rpc-indexer"]
     docker-org         = "autonomys"
-    docker-tag         = "mainnet-2025-jan-14"
```

Comment:
Lets not update mainnet bootstrap nodes yet. We still want them on Jan-14 release